plugins/reolink/src/main.ts

```diff
@@ -521,6 +596,7 @@ class ReolinkProvider extends RtspProvider {
             ScryptedInterface.Camera,
             ScryptedInterface.AudioSensor,
             ScryptedInterface.MotionSensor,
+            ScryptedInterface.DeviceProvider,
```

Comment:
don't add this interface unless it has a siren.

Reply:
Got it.  Updated to push the DeviceProvider interface when the siren ability is determined.